community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/AbstractBaseRecord.java

```diff
@@ -31,6 +31,9 @@
 public abstract class AbstractBaseRecord implements CloneableInPublic
 {
     private long id;
+    // Used for the "record unit" feature where one logical record may span two physical records,
+    // as to still keep low and fixed record size, but support occasionally bigger records.
+    private long secondaryId;
```

Comment:
this field is used only in dynamic format, would it be more reasonable to move it to specific subclass of AbstractBaseRecord which will be specific for that format?